js/main.js

```diff
@@ -33,4 +36,46 @@ function getRandomNumber(collectionLength) {
   return integer;
 }
 
+function handleClick(event) {
+  if (event.target.matches('.fa-solid')) {
+    var collectionData = {
+      collectionName: data.NFTvisible[0].collectionName,
+      collectionPhoto: null,
+      likes: null,
+      dislikes: null
+    };
+
+    var singleNFTData = {
+      name: data.NFTvisible[0].name,
+      data: data.NFTvisible[0].image
+    };
+
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', 'https://api.opensea.io/api/v1/collection/' + collectionData.collectionName.toLowerCase());
+    xhr.responseType = 'json';
+    xhr.addEventListener('load', function () {
+      collectionData.collectionPhoto = xhr.response.collection.image_url;
+    });
+    xhr.send();
+
+    if (Object.entries(data.ratingsInfo).length === 0) {
+      data.ratingsInfo[collectionData.collectionName] = collectionData;
+    }
+
+    for (var key in data.ratingsInfo) {
+      if (key !== data.NFTvisible[0].collectionName) {
```

Comment:
What is the purpose of this condition? I'm not sure if this is necessary

Reply:
im future proofing. probably went a little too far ahead. it's going to be going through the properties of ratingsInfo (which will be the names of the various collections).

if the collection name is already a property, then it won't add another with the same name. itll just increment the like property of the object that the collection name is storing.